declarative_clusterconfig/managing-app-sets-in-non-control-plane-namespaces.adoc

```diff
@@ -0,0 +1,44 @@
+:_mod-docs-content-type: ASSEMBLY   
+include::_attributes/common-attributes.adoc[]
+[id="managing-app-sets-in-non-control-plane-namespaces"]
+= Managing the application set resources in non-control plane namespaces
+:context: managing-app-sets-in-non-control-plane-namespaces
+
+toc::[]
+
+:FeatureName: `applicationSet` resources in non-control plane namespaces
+include::snippets/technology-preview.adoc[]
+
+By using application sets, you can automate and manage the deployments of multiple Argo CD applications declaratively from a single mono-repository to many clusters at once with greater flexibility.
+
+
+With {gitops-title} 1.12 and later, you can manage the `applicationSet` resources in non-control plane namespaces by explicitly enabling and configuring the Argo CD and `applicationSet` custom resources (CRs) as per your requirements. This functionality is particularly useful in multitenancy environments when you want to manage deployments of Argo CD applications for your isolated teams.
```

Comment:
Are back ticks required for applicationset CR references? Can we refer it the same way we refer ArgoCD CR?

cc: @svghadi

Reply:
I have left a suggestion to modify the sentence slightly. The ArgoCD & ApplicationSet are CRs so to highlight them I think we should use `` .